packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -612,6 +612,7 @@
   "Give_the_application_a_name_This_will_be_seen_by_your_users": "Give the application a name. This will be seen by your users.",
   "Global": "Global",
   "GoogleCloudStorage": "Google Cloud Storage",
+  "GoogleNaturalLanguage_ServiceAccount_Description": "Service account key JSON file. More information can be found [here](https://cloud.google.com/natural-language/docs/common/auth#set_up_a_service_account)",
```

Comment:
I think we'll eventually need to standardize how we word that. I have used different wording in Google Vision package: `Create a server key (JSON format) and paste the JSON content here` (yours seems better though)